src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="page">
+      <div class="calendar calendar--month-length--29">
+        <div class="calendar__item calendar__item--start-day--sun"></div>
```

Comment:
i think the calendar__item--start-day--sun modificator should be in block calendar